docs/2.0/integration/testng.adoc

```diff
@@ -1,29 +1,41 @@
-= TestNG
+=== TestNG
 An Allure report can be generated for any TestNG test. In order to get test results, you need to:
 
-* Add AllureTestListener to TestNG settings.
-* Add AspectJ Weaver dependency and its properties.
+* Download and uzip latest **allure-command-line**: https://bintray.com/qameta/generic/download_file?file_path=io%2Fqameta%2Fallure%2Fallure%2F2.0-BETA5%2Fallure-2.0-BETA5.zip
```

Comment:
you can wrap the link, https://dl.bintray.com/qameta/generic/io/qameta/allure/allure/2.0-BETA5/allure-2.0-BETA5.zip [**allure-commandline**]